contrib/ipmitool/patches/fix-enterprise-numbers.patch

```diff
@@ -0,0 +1,49 @@
+commit 324f434
+Author: Renato Botelho do Couto <renato@netgate.com>
+Date:   Sun Jun 30 21:13:09 2024
+
+    Makefile.am: enterprise-numbers will be always there
+
+diff --git a/ipmitool/Makefile.am b/ipmitool/Makefile.am
+index ce3267f..93daabc 100644
+--- a/ipmitool/Makefile.am
++++ b/ipmitool/Makefile.am
+@@ -49,36 +49,13 @@ dist-hook:
+ .PHONY: install-pen-database uninstall-pen-database
+ .INTERMEDIATE: %.o %.la enterprise-numbers
+ 
+-if DOWNLOAD
+-
+-enterprise-numbers:
+-	@echo Downloading IANA PEN database...
+-	@$(DOWNLOAD) "$(IANA_PEN)" > tmpfile.$$PPID || {\
+-		echo "FAILED to download the IANA PEN database"; \
+-		rm tmpfile.$$PPID; \
+-		false; \
+-	}
+-	@mv tmpfile.$$PPID $@
+-
+-install-pen-database: enterprise-numbers
++install-pen-database:
+ 	mkdir -m 755 -p $(DESTDIR)$(IANADIR)
+-	$(INSTALL_DATA) $< $(DESTDIR)$(IANADIR)/
++	$(INSTALL_DATA) enterprise-numbers $(DESTDIR)$(IANADIR)/
+ 
+ uninstall-pen-database:
```

Comment:
you can make this patch shorter by just replacing `if DOWNLOAD` with `if 1`, since the enterprise-numbers: rule will always be 'made' with the file being there, and then the rest just works